projects/storefrontstyles/scss/components/misc/spinner/_spinner-overlay.scss

```diff
@@ -0,0 +1,19 @@
+.is-loading {
+  position: relative;
+
+  .loader-overlay {
```

Comment:
very nice. I'd make it a first class citizen of the spinner, so that you can do:

```html
<cx-spinner class="overlay"></cx-spinner>
```

Reply:
so, you could move it to the spinner css